### PR TITLE
T#374/emails

### PR DESCRIPTION
**Pull request by sudo-may**

### CODE REVIEW

File: dashboard/src/components/header/HelpButton.vue [hunk 1]
```diff
@@ -16,21 +16,21 @@ limitations under the License.
 <template>
   <b-dropdown right variant="link" aria-label="Help Menu Dropdown" id="helpMenu">
     <template slot="button-content">
-      <b-avatar icon="question" size="sm" aria-hidden="true" variant="primary"></b-avatar>
+      <i class="far fa-question-circle" style="font-size: 1.55rem"></i>
```
Comment (mosefer) [inline dashboard/src/components/header/HelpButton.vue:19]: missing aria-hidden

File: dashboard/src/components/header/SettingsButton.vue [hunk 2]
```diff
@@ -16,28 +16,28 @@ limitations under the License.
 <template>
   <b-dropdown right variant="link" aria-label="User Settings Dropdown">
     <template slot="button-content">
-      <b-avatar variant="primary" size="sm" aria-hidden="true"></b-avatar>
+      <i class="fas fa-user-circle" style="font-size: 1.55rem"/>
```
Comment (mosefer) [inline dashboard/src/components/header/SettingsButton.vue:19]: missing aria-hidden

File: dashboard/src/components/settings/Preferences.vue [hunk 3]
```diff
@@ -20,10 +20,10 @@ limitations under the License.
     <loading-container v-bind:is-loading="isLoading">
         <div class="card">
           <div class="card-body">
-            <div class="mt-4">
+            <div class="">
               <b-form-group label-for="home-page-pref">
                 <template v-slot:label>
-                  Default Home Page:
+                  <i class="fas fa-home"></i> Default Home Page:
```
Comment (mosefer) [inline dashboard/src/components/settings/Preferences.vue:26]: missing aria-hidden

File: dashboard/src/components/skills/selfReport/SelfReportStatusPage.vue [hunk 4]
```diff
@@ -19,6 +19,9 @@ limitations under the License.
     <skills-spinner :is-loading="loading" />
     <div v-if="!loading">
       <self-report-info-cards :self-report-stats="selfReportStats"/>
+      <div v-if="showEmailServiceWarning" class="alert alert-warning mt-2" data-cy="selfReport_emailServiceWarning">
+        <i class="fa fa-exclamation-triangle" /> Please note that email notifications are currently disabled. Email configuration has not been performed on this instance of SkillTree. Please contact the root administrator.
```
Comment (mosefer) [inline dashboard/src/components/skills/selfReport/SelfReportStatusPage.vue:23]: add aria-hidden attribute to fa-exclamation-triangle please

File: service/src/main/java/skills/services/SelfReportingService.groovy [hunk 5]
```diff
@@ -117,14 +123,22 @@ class SelfReportingService {
...
-                skillDef: skillDefinition,
-                publicUrl: publicUrl,
-                projectName: projDef.name
-        ).build()
+                type: Notification.Type.SkillApprovalRequested.toString(),
+                keyValParams: [
+                        userRequesting: userAttrs.userIdForDisplay,
+                        numPoints     : skillDefinition.pointIncrement,
+                        skillName     : skillDefinition.name,
+                        approveUrl    : "${publicUrl}administrator/projects/${skillDefinition.projectId}/self-report",
```
Comment (mosefer) [inline service/src/main/java/skills/services/SelfReportingService.groovy:134]: do we guarantee anywhere that publicUrl ends with a trailing slash?

**Review by mosefer — CHANGES_REQUESTED** [review]

File: service/src/main/java/skills/services/SelfReportingService.groovy (hunk 5 above)
Comment (sudo-may) [reply service/src/main/java/skills/services/SelfReportingService.groovy:134]: @mosefer yes in ``skills.services.SelfReportingService#getPublicUrl``

**Review by mosefer — APPROVED** [review]

**Pull request merged**